### PR TITLE
add loop ggZHH and VHH NLO processes

### PR DESCRIPTION
**Pull request by chayanit**

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VHH_NLO/WHH_NLO_CV_1_0_C2V_1_0_C3_1_0_13TeV-madgraph/WHH_NLO_CV_1_0_C2V_1_0_C3_1_0_13TeV-madgraph_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+#***********************************************************************
+  tag_1     = run_tag ! name of the run 
+#***********************************************************************
+# Number of LHE events (and their normalization) and the required      *
+# (relative) accuracy on the Xsec.                                     *
+# These values are ignored for fixed order runs                        *
+#***********************************************************************
+ 100 = nevents ! Number of unweighted events requested 
+ 0.001 = req_acc ! Required accuracy (-1=auto determined from nevents)
+ 10000 = nevt_job! Max number of events per job in event generation.
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VHH_NLO/WHH_NLO_CV_1_0_C2V_1_0_C3_1_0_13TeV-madgraph/WHH_NLO_CV_1_0_C2V_1_0_C3_1_0_13TeV-madgraph_run_card.dat:33]: 2000 = nevents ! Number of unweighted events requested 
 0.001 = req_acc ! Required accuracy (-1=auto determined from nevents)
 -1 = nevt_job! Max number of events per job in event generation. 

Please change like above

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VHH_NLO/ZHH_NLO_CV_1_0_C2V_1_0_C3_1_0_13TeV-madgraph/ZHH_NLO_CV_1_0_C2V_1_0_C3_1_0_13TeV-madgraph_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+#***********************************************************************
+  tag_1     = run_tag ! name of the run 
+#***********************************************************************
+# Number of LHE events (and their normalization) and the required      *
+# (relative) accuracy on the Xsec.                                     *
+# These values are ignored for fixed order runs                        *
+#***********************************************************************
+ 100 = nevents ! Number of unweighted events requested 
+ 0.001 = req_acc ! Required accuracy (-1=auto determined from nevents)
+ 10000 = nevt_job! Max number of events per job in event generation.
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VHH_NLO/ZHH_NLO_CV_1_0_C2V_1_0_C3_1_0_13TeV-madgraph/ZHH_NLO_CV_1_0_C2V_1_0_C3_1_0_13TeV-madgraph_run_card.dat:33]: Same as above

**Comment by agrohsje** [comment]
Thanks @chayanit . I did minor modifications in case we use this as basline for multipartons. No impact on your request. 
@sihyunjeon looks good to me. Please merge if you have no further corrections.

**Comment by chayanit** [comment]
thanks @agrohsje!

**Comment by sihyunjeon** [comment]
Hi just modified one thing (which doesn't affect your physics targets).
Somewhere in MG releases between v265 and v310, authors made a crash exit so that users do not abuse [QCD] (automatically switching to loop-induced mode).
So users should specifically write [noborn=QCD] instead of [QCD]. I checked in MGv265 and v310 that the diagrams are same. Modified it for future reference.

**Pull request merged**